### ISSUE TITLE
Problem with gulp.watch example on home screen

### ISSUE THREAD

**Issue by johnnyreilly** [gulpjs/gulp#839]
This might me being obtuse but I as I've been having a play with ```gulp.watch``` I think I might have noticed a problem with https://github.com/gulpjs/gulp#sample-gulpfilejs.

Namely this:

```
// Rerun the task when a file changes
gulp.task('watch', function() {
  gulp.watch(paths.scripts, ['scripts']);
  gulp.watch(paths.images, ['images']);
});
```

Both ```scripts``` and ```images``` have ```clean``` as a dependency which deletes the current build.  Presumably with the approach documented here the amending of a script would leave you with a build folder without any images and the changing of an image would leave you with a build folder without any scripts?

For what it's worth I seem to be hit hard by the file watching issues detailed in https://github.com/gulpjs/gulp/issues/651.  But thought I should raise this possible documentation issue in case it misleads.  (Though if I've got the wrong end of the stick then do set me straight!)

**Comment by jalada** [comment]
Why is this README example also in the repository for the tag 3.8.10? https://github.com/gulpjs/gulp/blob/v3.8.10/README.md

**Comment by heikki** [comment]
Fixed in 4.0 branch: https://github.com/gulpjs/gulp/blob/4.0/README.md#sample-gulpfilejs
